internal/promotion/runner/builtin/git_pr_opener.go

```diff
@@ -159,6 +164,7 @@ func (g *gitPROpener) run(
 			Status: kargoapi.PromotionPhaseSucceeded,
 			Output: map[string]any{
 				stateKeyPRNumber: pr.Number,
+				stateKeyRepoURL:  cfg.RepoURL,
```

Comment:
This isn't the right information to add. You want `pr.URL` and the symbol and value of the key should probably be changed as well on (new) line 29:

```
	stateKeyPRURL = "prURL"
```

Reply:
Ah. I see this was already discussed in #3450.